core/trino-main/src/main/java/io/trino/execution/scheduler/StageScheduler.java

```diff
@@ -18,6 +18,8 @@
 public interface StageScheduler
         extends Closeable
 {
+    default void start() {}
```

Comment:
Maybe add a short documentation, e.g.:

`Called by the query scheduler when the scheduling process begins. This method is called before the ExecutionSchedule takes a decision to schedule a stage but after the query scheduling has been fully initialized. Within this method the scheduler may decide to schedule tasks that are necessary for query execution to make progress. For example the the scheduler may decided to schedule a task without assigning any splits to unblock dynamic filter collection.`